styleguide/components/components.scss

```diff
@@ -1 +1,3 @@
+import '_alerts.scss';
+import '_buttons.scss';
```

Comment:
When using import, SASS is smart and knows what file you're referring to if you omit the .scss  and the _

In SASS, the underscore _ in front of a file name is more of an organization thing so it indicates to the developer that it is a "partial" or a file that is being included or imported into another file. So you can change these imports to this and it will work:
```
import 'alerts';
import 'buttons';
```